run_model.py

```diff
@@ -61,6 +70,12 @@ def run_state(state=None, deploy=False):
     run.run_state_level_forecast(
         min_date, max_date, country="USA", state=state, output_dir=output_dir
     )
+    logging.info(f'Wrote output to {output_dir}')
```

Comment:
Would prefer if we used a module level logger to give a bit more context to where the log is coming from.

Reply:
Done.